Orange/widgets/data/owconcatenate.py

```diff
@@ -64,7 +64,7 @@ def __init__(self):
         self.primary_data = None
         self.more_data = OrderedDict()
 
-        mergebox = gui.vBox(self.controlArea, "Domain Merging")
+        self.mergebox = mergebox = gui.vBox(self.controlArea, "Domain Merging")
```

Comment:
`mergebox` is only used once, so instead of defining it, you could just use `self.mergebox` in L69.
Simpler and in this case even shorter.